docusaurus/docs/cms/api/client.md

```diff
@@ -67,9 +83,11 @@ If you're using the Strapi Client in a browser environment, you can include it u
 </script>
 ```
 
+The `baseURL` must include the protocol (`http` or `https`). An invalid URL will throw an error `StrapiInitializationError`.
+
 ### Authentication
 
-The Strapi Client supports different authentication strategies to access protected resources in your Strapi back end.
+The Strapi Client currently only supports API tokens as an authentication method, to access protected resources in your Strapi back end.
```

Comment:
```suggestion
The Strapi Client currently only supports [API tokens](/cms/features/api-tokens) as an authentication method, to access protected resources in your Strapi back end.
```

If the content is validated by devs, it's better to add a link to the related docs (we try to cross-reference docs as much as possible 🤓 ).